typescript-id-allocator/IdCompressor.ts

```diff
@@ -61,12 +60,12 @@ export class IdCompressor implements IIdCompressor, IIdCompressorCore {
 		if (wasmRange.ids === undefined) {
 			range = { sessionId: this.localSessionId };
 		} else {
-			const { first_local, count } = wasmRange.ids;
```

Comment:
This wasn't building.

Reply:
Yep, missed that. Thanks.

Reply:
I see you also updated the external types to reflect the gencount change. Good stuff.